sqlglot/generator.py

```diff
@@ -741,7 +741,7 @@ def maybe_comment(
 
         return f"{sql} {comments_sql}"
 
-    def wrap(self, expression: exp.Expression | str) -> str:
+    def wrap(self, expression: exp.Expression | str, left="(", right=")") -> str:
```

Comment:
Let's not touch `wrap` for this PR, it shouldn't be relevant.

Reply:
I couldn't came anything better. This function adds extra (), not sure yet how to fix it without touching wrap function. Maybe you could help?